charts/galley/templates/configmap.yaml

```diff
@@ -62,5 +62,6 @@ data:
         sso: {{ .settings.featureFlags.sso }}
         legalhold: {{ .settings.featureFlags.legalhold }}
         teamSearchVisibility: {{ .settings.featureFlags.teamSearchVisibility }}
+        classifiedDomains: {{ .settings.featureFlags.classifiedDomains }}
```

Comment:
Ah, I found the issue you have. This renders as

```
      featureFlags:
        sso: disabled-by-default
        legalhold: whitelist-teams-and-implicit-consent
        teamSearchVisibility: disabled-by-default
        classifiedDomains: map[config:map[domains:[example.com]] status:enabled]
---
```

so you need to use another helm magic function to get this to render properly. See https://stackoverflow.com/questions/63619667/helm-include-formatted-map-from-values-yml

test it out using:

```diff
--- a/hack/bin/integration-setup.sh
+++ b/hack/bin/integration-setup.sh
@@ -4,22 +4,22 @@ USAGE="Usage: $0"

-charts=( fake-aws databases-ephemeral wire-server nginx-ingress-controller nginx-ingress-services )
+charts=( wire-server )
 
 echo "Installing charts..."
@@ -69,6 +69,8 @@ for chart in "${charts[@]}"; do
         --set brig.config.optSettings.setFederationDomain="$FEDERATION_DOMAIN" \
         --set galley.config.settings.federationDomain="$FEDERATION_DOMAIN" \
         --wait \
+        --debug \
+        --dry-run \
         --timeout "$TIMEOUT"
     set +x
 done

-until resourcesReady; do echo 'waiting for SNS resources'; sleep 1; done
-
-kubectl -n ${NAMESPACE} get pods
 ```
 
 then running `DOCKER_TAG=2.109.27 make kube-integration-setup-sans-federation`

Reply:
This has been taken care of.